DigitalPassaport/DigitalPassaport/Epics/FlashPass/Repository/FlashPassRepository.swift

```diff
@@ -20,7 +20,7 @@ class FlashPassRepositoryImpl: FlashPassRepository {
     
     func getUser() -> User {
```

Comment:
Also when we use database we should fetch data in async way, so here we should return AnyPublisher if we use combine, or async/await if use in that way. Maybe you should refactor you database to return some publisher, not just value